docs/configure

```diff
@@ -492,7 +487,6 @@ dnslookup:
   domains = ! +local_domains
   transport = remote_smtp
   ignore_target_hosts = 0.0.0.0 : 127.0.0.0/8
-  headers_remove = X-Spam-Report:X-Spam-Score
```

Comment:
>> For this reason, I currently use this header to set the filter:
> X-Spam-Report: Spam detection software, running on the system "example.org",
> the X-Spam-Report-tag in combination with example.org is unique for my mail server. I am not happy with that at all.

> Why's this bad?

It's working, however you have to be careful about this rules. Moving the server, I had to change the host name and such settings are very likely to be missed.




> After I migrated my setup to 2.3 couple days ago, my spam filtering became worse. And I don't even know how to start fixing that...

That is bad. You wrote in a different post, that you run sa-exim before, you are sure that you run spamassassin as the same user with the same settings? Or you had more ACL-checks in use which are not part of the current vexim setup (SPF, rDNS checks, ...)?